packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -299,8 +298,13 @@ $btn-icon-sizeXSmall-buttonSize        : 32px;
     outline: 0;
 
     // Show focus styles on keyboard focus.
+    &:focus,
     &:focus-visible {
-        box-shadow: 0 0 0 2px $btn-default-outline-color;
+        @extend %u-elementFocus;
+
+        &, &:after {
```

Comment:
I think you've got this as a TODO by the looks of it, but would be good to add a silent extend for this as looks like it's used in quite a few places.

Reply:
Yep, I can add that in the next PR?

Reply:
Ideally I'd have it as some sort of mixin where you can specify what you want the radius to be (similar to what we do with spacing or font size), what do you reckon?

Reply:
Yeh, I think that sounds good 👍